ci/check_line_lengths.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+echo "Checking line lengths in all source files <= $MAX_LINE_LENGTH chars..."
+
+echo "Offending files and lines:"
+(( success = 1 ))
+for file in src/*.md ; do
```

Comment:
I think we also need to check subdirectories (e.g. `test`)

Reply:
Yeah. We have a problem though: in tables, we can't just wrap at 80 chars. Hrmm.